chain-impl-mockchain/doc/format.abnf

```diff
@@ -139,6 +145,46 @@ CHANGES           = SIZE-ELEMENT-16BIT ; number of changes
 
 PROPOSAL-ID       = FRAGMENT-ID
 
+; ####################
+; FRAGMENT Vote plan, vote cast, vote tally
+; ####################
+
+VOTE-PLAN         = PLAN-CERT IOW BFT-SIGNATURE
```

Comment:
1) technically this is not a bft signature, although in practice they are equal
2) it also contains the id of the committee member that is signing the transaction

Reply:
> it also contains the id of the committee member that is signing the transaction

Is it not a part of the `ED25519-SIGNATURE` ? 
I thought it should corresponds to the following 
```
UPDATE-PROPOSAL   = PROPOSAL-CERT IOW BFT-SIGNATURE
UPDATE-VOTE       = VOTE-CERT IOW BFT-SIGNATURE
```

Reply:
I think it's not, `BTF-SIGNATURE` is just 64 bytes of signature. Look at how this is used in https://github.com/input-output-hk/chain-libs/blob/31fafb407ac2d0b519ed297410c1d128ff510f23/chain-impl-mockchain/doc/format.abnf#L20

We might need to fix `UPDATE-{PROPOSAL, VOTE}` if that's the case